docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -340,6 +340,8 @@ Noteworthy module changes
 Plugins
 =======
 
+* macOS no longer defaults to using the ``paramiko`` connection plugin as it has since Ansible 1.8. It will now use the ``ssh`` connection plugin by default.  Since ``ssh`` supports connection persistence between tasks and playbook runs, it performs better than ``paramiko``. If you are using password authentication, you will need to install ``sshpass`` when using the ``ssh`` connection plugin. Or you can explicitly set the connection type to ``paramiko`` to maintain the pre-2.8 behavior on macOS.
```

Comment:
I'm confused about the timeline and versions here - is this a change in how macOS functions, or in how Ansible functions? If it's macOS, I'd say "as it has since OSX <nickname>"; if it's Ansible, I'd say "Ansible no longer defaults to the `paramiko` connection plugin on control nodes running macOS" (and fill in the other half of the sentence in each case).

Reply:
Good point. This is why I like to have peer review. Now I see how it can be interpreted both ways. I will change it so that it's clear it is a change in Ansible on a macOS control node.